client/app/components/items-list/components/ItemsTable.jsx

```diff
@@ -136,10 +139,13 @@ export default class ItemsTable extends React.Component {
       })
     ) : null;
 
+    const { showHeader } = this.props;
+
     return (
       <Table
-        className="table-data"
+        className={classNames('table-data', { 'ant-table-headerless': !showHeader })}
```

Comment:
Hmmm.. I think that's risky to rely on Ant's implementation like this.

Reply:
@ranbena what do you mean?

Reply:
Use the built in `<Table showHeader={false} />` prop instead.

Reply:
@ranbena  I already use it (see few lines below). `ant-table-headerless` is a custom class - to add top border line for first table row when header is invisible (by default there is no border) - https://github.com/getredash/redash/pull/3411/files#diff-8afda93fc5d4b6826361faf186019212R187

Reply:
So is it an Ant bug? Sounds like "ant-table-headerless" should be applied by default when `showHeader={false}`.

The risk is that the ant classnames are internal for Ant, which means they can change with versions. We can't rely on them.

Reply:
That's not Ant class - it's custom (also defined in this PR - see the link). I decided to prefix it with `ant-table-` because it's applicable only to Ant's tables, that's it.

Reply:
I'm sorry man, my mistake🙇‍♂️ Good job!

Reply:
NP :+1: